src/libraries/Microsoft.PowerFx.Interpreter/CustomFunction/ReflectionFunction.cs

```diff
@@ -269,6 +269,10 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
                 {
                     arg = FormulaValue.New(string.Empty);
                 }
+                else if (arg is BlankValue && expectedType is RecordType)
+                {
+                    // Add arg as is.
+                }

```

Comment:
Adding this to avoid breaking changes to existing custom functions by other hosts.
Ideally a `BlankValue` should be able to hit runtime implementation.
I'll gather more info on how this might affect hosts.

Reply:
I think this is fine. Blank is a valid case and so we shouldn't fail, like the other cases here. 

Just add a test for it.